clustered/server/src/main/java/org/ehcache/clustered/server/management/Management.java

```diff
@@ -24,60 +24,43 @@
 import org.terracotta.entity.ClientDescriptor;
 import org.terracotta.entity.ServiceRegistry;
 import org.terracotta.management.model.context.Context;
+import org.terracotta.management.registry.collect.StatisticConfiguration;
 import org.terracotta.management.service.monitoring.ConsumerManagementRegistry;
 import org.terracotta.management.service.monitoring.ConsumerManagementRegistryConfiguration;
 import org.terracotta.management.service.monitoring.registry.provider.ClientBinding;
-import org.terracotta.offheapresource.OffHeapResource;
-import org.terracotta.offheapresource.OffHeapResourceIdentifier;
 
-import java.util.Set;
-import java.util.concurrent.Executors;
-import java.util.concurrent.ScheduledExecutorService;
-import java.util.concurrent.atomic.AtomicLong;
+import static java.util.concurrent.TimeUnit.SECONDS;
 
 public class Management {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(Management.class);
 
-  // TODO FIXME: the following things are just temporary and should be removed/changed asap
-  // - scheduling should be done by using a voltron service (not yet available: see https://github.com/Terracotta-OSS/terracotta-apis/issues/158)
-  // - stats config should be given when configuring the entities (https://github.com/ehcache/ehcache3/issues/1567)
-  private static final AtomicLong managementSchedulerCount = new AtomicLong();
-  private ScheduledExecutorService managementScheduler;
-  private final StatisticConfiguration statisticConfiguration = new StatisticConfiguration();
+  // TODO: if a day we want to make that configurable, we can, and per provider, or globally as it is now
+  private final StatisticConfiguration statisticConfiguration = new StatisticConfiguration(
```

Comment:
this config remains... It controls the specific ehcache entity stats. It is hard coded at the moment and cannot be changed, but perhaps one day we would want to have it inside the ehcache entity configuration part of the xml ?